curation-migration-datasync/eventConsumer.integration.ts

```diff
@@ -0,0 +1,250 @@
+import { CuratedItemRecord, ScheduledSurfaceGuid } from './dynamodb/types';
```

Comment:
this is almost a copy-paste from index.integration.ts

I added a nested describe so that the 'add' is separated from other events, and scoped the test variables/assertions there. I used the eventConsumer method instead of handlerFn